meta-lmp-base/recipes-samples/images/lmp-feature-alsa.inc

```diff
@@ -0,0 +1,5 @@
+# Basic ALSA packages
+CORE_IMAGE_BASE_INSTALL += " \
+    alsa-utils \
+    alsa-state \
+"
```

Comment:
I think these are already installed by default once alsa is available in DISTRO_FEATURES:

```
oe-core/meta/recipes-core/packagegroups/packagegroup-base.bb:

SUMMARY:packagegroup-base-alsa = "ALSA sound support"
RDEPENDS:packagegroup-base-alsa = "\
    alsa-utils-alsactl \
    alsa-utils-amixer \
    ${VIRTUAL-RUNTIME_alsa-state}"
...
RDEPENDS:packagegroup-base = "\
    packagegroup-distro-base \
    packagegroup-machine-base \
    \
    module-init-tools \
    ${@bb.utils.contains('MACHINE_FEATURES', 'apm', 'packagegroup-base-apm', '',d)} \
    ${@bb.utils.contains('MACHINE_FEATURES', 'acpi', 'packagegroup-base-acpi', '',d)} \
    ${@bb.utils.contains('MACHINE_FEATURES', 'keyboard', 'packagegroup-base-keyboard', '',d)} \
    ${@bb.utils.contains('MACHINE_FEATURES', 'phone', 'packagegroup-base-phone', '',d)} \
    \
    ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', 'packagegroup-base-alsa', '',d)} \

```

Reply:
@ricardosalveti Not really. Only `alsa-state` and 2 utils `alsactl` and `amixer` are installed with the MACHINE_ or COMBINED_FEATURES alsa. Neither aplay/arecord nor test sounds, sample configs etc are installed with that feature got set. So I'm going to use COMBINED_FEATURES instead of DISTRO_FEATURES but we still need to include the full `alsa-utils` package, not just some subpackages.

Reply:
Sure, but in this case we can safely drop at least alsa-state, right? Since packagegroup-base-alsa will be installed anyway.

Reply:
Done.